contracts/FlowServiceAccount.cdc

```diff
@@ -185,6 +186,11 @@ access(all) contract FlowServiceAccount {
                 emit IsAccountCreationRestrictedUpdated(isRestricted: enabled)
             }
         }
+
+        access(all)
+        fun governanceDirectCall(from: String, to: String, amount: UInt): EVM.Result {
+            return EVM.reclaimFundsFromAttackerEOAs(from: from, to: to, amount: amount)
```

Comment:
`reclaimFundsFromAttackerEOAs` is `access(account)` (https://github.com/onflow/flow-go/pull/8280/commits/dde3be6f718a325f73c9ab0061e0d962ee60010a#diff-02837edc597afad5bd8f7d1962c129bded25adcc21a9889382c7aaad9a0e5b34R1035). Are the `EVM` contract and the `FlowServiceAccount` contract deployed to the same account?